HHS/Head-Start-TTADP
# [TTAHUB-1977] Change urls for training reports

## Description of change

Given a smartsheet provided ID for a training report, update training report urls to be /training-report/XXXX where X is the trailing numbers of that smartsheet ID. (Example: R01-TR-1234 would have a URL of /training-report/1234)

Also update session URLs to match: session with a DB id of 300 would have a URL of /training-report/1234/session/300

## How to test

- Ensure all CRUD operations for the training and session reports still work
- Inspect the database to confirm that no additional reports are being created
- Verify the URLs conform to the new design

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1977

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
